CHANGELOG.md

```diff
@@ -6,10 +6,77 @@
 
 #### New Features
 
+- Added support for 'Service' domain to `session.lineage.trace` API.
```

Comment:
why change this file in this PR?

Reply:
Merging changes to `main` requires updating the changelog. However, it's unclear to me why this change made it here. I'll need to see how the commit chain got applied - this isn't correct here.